src/common/search/SearchService.js

```diff
@@ -229,6 +229,15 @@
       mapService_.map.addLayer(searchlayer_);
       forEachArrayish(results, function(result) {
         var olFeature = new ol.Feature();
+        olFeature.properties = result;
+        if (result.name.length > 25) {
+          // id is set twice to be handled correctly by featureInfoBox service
+          olFeature.id = result.name.substring(0, 25) + '...';
+          olFeature.setId(result.name.substring(0, 25) + '...');
```

Comment:
The private member is normally used, but for some reason the public "id" is also used by the feature manager service (cannot find the code, will update comment with link)